### PR TITLE
Extend webpack config with font extentions

### PR DESCRIPTION
**Pull request by lotrien**
It was done to give the ability to use icon font within this project

### CODE REVIEW

**Review by ikalnytskyi — APPROVED** [review]

File: webpack.config.js [hunk 1]
```diff
@@ -40,6 +40,17 @@ module.exports = () => {
           use: ['babel-loader'],
         },
 
+        {
+          test: /\.(woff|svg|ttf|woff2|eot)$/,
+          include: path.resolve(__dirname, 'src/fonts'),
```
Comment (ikalnytskyi) [inline webpack.config.js]: Could you please do not use `/` separator and just replace it with

```
path.resolve(__dirname, 'src', 'fonts')
```

File: webpack.config.js [hunk 2]
```diff
@@ -50,7 +61,7 @@ module.exports = () => {
               {
                 loader: 'css-loader',
                 options: {
-                  modules: true,
+                  modules: false,
```
Comment (ikalnytskyi) [inline webpack.config.js]: Could you please move it into separate PR?

**Review by ikalnytskyi — CHANGES_REQUESTED** [review]

File: webpack.config.js (hunk 2 above)
Comment (malor) [inline webpack.config.js]: Ok, so my understanding is that this is https://github.com/css-modules/css-modules . I didn't even know we could have scopes in CSS :P

File: webpack.config.js (hunk 1 above)
Comment (lotrien) [reply webpack.config.js]: fixed

File: webpack.config.js (hunk 2 above)
Comment (lotrien) [reply webpack.config.js]: reverted from this pr

**Review by malor — DISMISSED** [review]
Custom fonts? Why not! 👍

File: webpack.config.js (hunk 2 above)
Comment (ikalnytskyi) [reply webpack.config.js]: Well, actually, CSS modules are webpack extension. :P

**Review by ikalnytskyi — APPROVED** [review]
